src/server/pfs/server/url.go

```diff
@@ -186,7 +186,7 @@ func (d *driver) getFileURL(ctx context.Context, taskService task.Service, URL s
 		err := doer.Do(ctx, inputChan, func(_ int64, output *types.Any, err error) error { return err })
 		return errors.EnsureStack(err)
 	})
-	var bytesWritten int64
+	var size int64
```

Comment:
This isn't correct. Bytes written is separate from the sharding criteria. We should revert this back to bytes written and collect it independent of the size variable. Also, we should rename count to numObjects for consistency.

Reply:
Good catch, size is getting reset, but bytesWritten shouldn't be.